package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "npm-run-all --parallel dev:server dev:client",
+    "dev:server": "convex dev",
+    "dev:client": "next dev",
```

Comment:
Now all we need to do is run `bun dev` locally and the next server and convex server will start up together.

Reply:
### Before

<img width="1087" alt="dev-before" src="https://github.com/kenneth-duverge/the-newsletter/assets/77157695/db04c05d-7816-406a-88e8-cd4c09171b50">

### After
<img width="850" alt="dev-after" src="https://github.com/kenneth-duverge/the-newsletter/assets/77157695/5971e75b-aac4-45a5-a491-e72041fd8365">